curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -200,6 +201,8 @@ MetaStatusCode InodeManager::GetXAttr(uint32_t fsId, uint64_t inodeId,
 
 MetaStatusCode InodeManager::DeleteInode(uint32_t fsId, uint64_t inodeId) {
     VLOG(6) << "DeleteInode, fsId = " << fsId << ", inodeId = " << inodeId;
+    InodeAttr attr;
```

Comment:
move it to lock range

Reply:
> move it to lock range

There is lock in GetInodeAttr, moving to lock range will cause deadlock